pymeasure/instruments/keithley/keithley2306.py

```diff
@@ -563,40 +552,21 @@ class Step():
     )
 
     def __init__(self, instrument, number):
-        self.instrument = instrument
-        self.number = number
-
-    def values(self, command, **kwargs):
-        return self.instrument.values(command.replace("<step>", str(self.number)),
-                                      **kwargs)
+        super().__init__(instrument, number)
```

Comment:
Shouldn't we add `**kwargs` for cooperative classes?

Reply:
good point.